README.md

```diff
@@ -0,0 +1,6 @@
+Hello, World
+============
+
+A simple program, and an example of how to structure a python project.
+You can invoke the program easily by typing "python main.py" or "python helloworld.py" in the helloworld folder.<br>
```

Comment:
I'd strongly prefer not to have 2 different recommended commands to run it.

My recommended instructions to run it would be:

1. `python setup.py install`
1. `helloworld_in_python`

because not all python programs can be run without installing them first and I'd like this example to be as general-purpose as possible.

If you'd also like to suggest a way to run it without installing, `./helloworld.py` would be the recommended way since that file is executable and has a python shebang line.